app/controllers/bookings_controller.rb

```diff
@@ -1,16 +1,43 @@
 class BookingsController < ApplicationController
+  before_action :set_bridge
+
   def index
+    @bookings = Booking.where(bridge_id: @bridge.id)
   end
 
   def show
+    @booking = Booking.find(params[:id])
   end
 
   def new
+    @booking = Booking.new
   end
 
   def create
+    @booking = Booking.new(booking_params)
+    @booking.user_id = current_user
+    @booking.bridge_id = @bridge.id
+    respond_to do |format|
+      if @booking.save
+        format.html { redirect_to [@bridge, @booking], notice: 'Congrats! Bridge booked.' }
+        format.json { render :show, status: :created, location: @booking }
+      else
+        format.html { render :new }
+        format.json { render json: @booking.errors, status: :unprocessable_entity }
+      end
+    end
   end
 
   def destroy
```

Comment:
we want to unable in the future a hobo to cancel a booking so I suggest we leave it for now and reassess later